### PR TITLE
Restoring tests

### LINKED ISSUE #1335: Add PKValuesCalculator back to code (commented out at the moment) because we could not import file from excel

### PR DESCRIPTION
**Pull request by abdelr**
Fixes #1335 (Restoring tests and making them independent from excel packages)

### CODE REVIEW

File: tests/OSPSuite.Core.IntegrationTests/OSPSuite.Core.IntegrationTests.csproj [hunk 1]
```diff
@@ -63,6 +63,9 @@
     <None Update="Data\EventsAppKeywordReplacement.pkml">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="Data\MultipleDosing_0_8_16.csv">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
     <None Update="Data\MultipleDosing_0_8_16.xls">
```
Comment (msevestre) [inline tests/OSPSuite.Core.IntegrationTests/OSPSuite.Core.IntegrationTests.csproj:69]: Let's remove the excel files if they are not used anymore

**Review by msevestre — COMMENTED** [review]
Looks good. Let's remove the excel file and we can merge

File: tests/OSPSuite.Core.IntegrationTests/OSPSuite.Core.IntegrationTests.csproj (hunk 1 above)
Comment (abdelr) [reply tests/OSPSuite.Core.IntegrationTests/OSPSuite.Core.IntegrationTests.csproj:69]: I did already ;)
https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/pull/1599/commits/04b49097a26bc91d2d1046b168da4e05c866551f#diff-941dec3a96d741e2508f082ae94091684e864546b347ecafdc39f3dca68af427
https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/pull/1599/commits/04b49097a26bc91d2d1046b168da4e05c866551f#diff-f0a484c7362aa86842f38acac2348d5cb163d20878540e3ab3becf6091d3664a

File: tests/OSPSuite.Core.IntegrationTests/OSPSuite.Core.IntegrationTests.csproj [hunk 2]
```diff
@@ -75,6 +78,9 @@
     <None Update="Data\SingleDosing.xls">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="Data\SingleDosing_CL.csv">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
     <None Update="Data\SingleDosing_CL.xls">
```
[the remark this answers is not in the document]
Comment (msevestre) [reply tests/OSPSuite.Core.IntegrationTests/OSPSuite.Core.IntegrationTests.csproj:84]:
```
 <None Update="Data\SingleDosing.xls">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </None>
    ```

[the remark this answers is not in the document]
Comment (abdelr) [reply tests/OSPSuite.Core.IntegrationTests/OSPSuite.Core.IntegrationTests.csproj]: You are right. I have no idea why. It was actually not listed in the solution. I removed them manually now.

**Pull request merged**